### PR TITLE
chore: upgrade husky 4 => 6; bump eslint 7.22 => 7.23;

### PR DESCRIPTION
**Pull request by a-ogilvie**
This PR upgrades husky v4 => v6. There is one breaking change: we must move husky configs from `husky.config.js` into `.husky/<name-of-hook>` when using husky v6.

For this reason, I'd like to bump `@zehitomo/eslint-config` to v4 once we merge this change.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,21 +24,25 @@
...
+    "husky": "6.0.0",
     "lint-staged": "10.5.4",
     "prettier": "2.2.1",
     "stylelint": "13.12.0"
   },
   "devDependencies": {
-    "eslint": "7.22.0",
-    "husky": "4.3.8",
+    "eslint": "7.23.0",
+    "husky": "^6.0.0",
```
Comment (dricholm) [inline package.json]: Should we pin this as before?

**Review by dricholm — APPROVED** [review]
Looks good.
One thing I noticed is when running `npm i` locally there was a change in `package-lock.json`, line 10 was removed: `"hasInstallScript": true,`, not sure why though and whether it would have any side effects.
Another question is whether to disable Husky on CI. The [official documentation](https://typicode.github.io/husky/#/?id=disable-hooks-in-ci) has some methods if we want to do it.

**Comment by a-ogilvie** [comment]
> One thing I noticed is when running `npm i` locally there was a change in `package-lock.json`, line 10 was removed: `"hasInstallScript": true,`, not sure why though and whether it would have any side effects.

I had originally set up the package to have a `postinstall` script. However, it seems that `prepare` will also work for us, and seems to be the recommended way to install.

> Another question is whether to disable Husky on CI. The [official documentation](https://typicode.github.io/husky/#/?id=disable-hooks-in-ci) has some methods if we want to do it.

We can disable Husky by setting a `HUSKY` env var on CircleCI to `0`. Let's do that if it causes any problems.

Husky won't be installed in the build steps anyway, because they don't install devDeps.

File: .gitignore [hunk 2]
```diff
@@ -1 +1,3 @@
 node_modules
+
+.vscode
```
Comment (a-ogilvie) [inline .gitignore:3]: I keep my preferences in this folder 😅

**Review by dricholm — APPROVED** [review]

**Pull request merged**